src/store/editor-slice.ts

```diff
@@ -83,7 +83,10 @@ function createEditorReducer(
 				signature.value.id === payload.settings.prefs.zimbraPrefForwardReplySignatureId
 		)?.value.description ?? '';
 
-	const empty = emptyEditor(payload.editorId, payload.accounts);
+	const empty = {
+		...emptyEditor(payload.editorId, payload.accounts),
+		text: textWithSignatureNewMessage
+	};
 	state.editors[payload.editorId] = empty;
 
```

Comment:
The only problem I found in this PR is that it's not rendering the Images of the signature

Reply:
Let's wait for more reviews. In my case it is working fine, I guess the problem could be in your endpoint